src/infra/vhost/virtualHostCmdRepo.go

```diff
@@ -506,6 +506,11 @@ func (repo VirtualHostCmdRepo) CreateMapping(createMapping dto.CreateMapping) er
 		locationContent += " " + createMapping.TargetUrl.String()
 	}
 
+	if createMapping.TargetType.String() == "inline-html" {
+		locationContent = "	add_header Content-Type text/html;\n" + locationContent
+		locationContent += " '" + createMapping.TargetInlineHtmlContent.String() + "'"
```

Comment:
Não tá faltando um ; no final aqui?

Reply:
Já existe, está mais abaixo. É o mesmo ";" que é utilizado para o "url" e "response-code".
Eu não mexeria nisso por enquanto, vai ser melhorado quando o SQLite for implementado.